### PR TITLE
completed javascript-if - brighitte johnson

### PR DESCRIPTION
**Pull request by brij8**
javascript-if

### CODE REVIEW

File: javascript-if/main.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  return num < 5;
+}
+
+function isEven(num) {
+  return (num % 2 === 0);
+}
+
+function startsWithJ(string) {
+  return string[0] === 'J' || string[0] === 'j';
+}
```
Comment (TimelikeClosure) [inline javascript-if/main.js:20]: While it's good to think about multiple versions of the same letter, this particular function specifies a _capital_ `'J'`:

---

image

File: javascript-if/main.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+function isOldEnoughToDrink(person) {
+  return person.age >= 21;
+}
+function isOldEnoughToDrive(person) {
+  return person.age >= 16;
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return 'false - this is illegal';
+}
```
Comment (TimelikeClosure) [inline javascript-if/main.js:31]: The `return` value should have a `Boolean` data type:

---

image

File: javascript-if/main.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      if (pH > 7 && pH < 14) {
+        return 'base';
+      } else {
+        if (pH < 0 || pH > 14) {
+          return 'invalid pH level';
+        }
+      }
+    }
+  }
+}
```
Comment (TimelikeClosure) [inline javascript-if/main.js:49]: What happens if `pH === 0` or `pH === 14`? Please make sure your function accounts for those possibilities:

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by brij8** [comment]
oop, didn't see there were multiple fixes.. on them now

**Comment by brij8** [comment]
javascript-if-01

File: javascript-if/main.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+function isOldEnoughToDrink(person) {
+  return person.age >= 21;
+}
+function isOldEnoughToDrive(person) {
+  return person.age >= 16;
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return person.age < 0;
+}
```
Comment (TimelikeClosure) [inline javascript-if/main.js:31]: Unborn children are allowed to? Wish I knew that.

**Review by TimelikeClosure — APPROVED** [review]

File: javascript-if/main.js (hunk 4 above)
Comment (brij8) [reply javascript-if/main.js:31]: **only** unborn children allowed
...had to get that false return somehow lol

**Pull request merged**